fs2-google-pubsub-http/src/main/scala/com/permutive/pubsub/consumer/http/internal/Model.scala

```diff
@@ -32,21 +37,44 @@ private[http] object Model {
     maxMessages: Int
   )
 
+  object PullRequest {
+    implicit final val PullRequestCodec: JsonValueCodec[PullRequest] =
+      JsonCodecMaker.make[PullRequest](CodecMakerConfig)
+  }
+
   case class PullResponse(
     receivedMessages: List[ReceivedMessage]
   )
 
+  object PullResponse {
+    implicit final val PullResponseCodec: JsonValueCodec[PullResponse] =
+      JsonCodecMaker.make[PullResponse](CodecMakerConfig)
+  }
+
   case class ReceivedMessage(
     ackId: AckId,
     message: PubsubMessage
   )
 
+  object ReceivedMessage {}
+
   case class AckRequest(
     ackIds: List[AckId]
   )
 
-  case class NackRequest(
+  object AckRequest {
+    implicit final val AckRequestCodec: JsonValueCodec[AckRequest] =
+      JsonCodecMaker.make[AckRequest](CodecMakerConfig)
+  }
+
+  case class ModifyAckDeadlineRequest(
     ackIds: List[AckId],
-    ackDeadlineSeconds: Int
+    ackDeadlineSeconds: Long
```

Comment:
This is definitely a `Long` on Google's end?

Reply:
The docs (https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.subscriptions/modifyAckDeadline) says `number` so yes? 👀

Reply:
Potentially true, but seeing as the maximum is `600` do we really need the `Long`? 😛 

This would be a classic case for the refined...